charts/cluster-api-runtime-extensions-nutanix/addons/ccm/aws/map-template.yaml

```diff
@@ -0,0 +1,7 @@
+# These values are a combination of a Helm template to create the dict for the k8s version to
+# CCM version map, and then including the non-templated values from the values-template.yaml.
+# This simplifies the templating required in the values-template.yaml.
+$k8sMinorVersionToCCMVersion := dict
+{{ range $k8sVersion, $ccmVersion := .Values.hooks.ccm.aws.k8sMinorVersionToCCMVersion -}}
+  "{{ $k8sVersion }}" "{{ $ccmVersion }}"
```

Comment:
not too familiar with the dict format. 
should the KV pair assigned to the `k8sMinorVersionToCCMVersion` ?
ie.
```
set $k8sMinorVersionToCCMVersion "{{ $k8sVersion }}" "{{ $ccmVersion }}"
```

Reply:
no it should be a dict of all the values of k8s minor to image version

```yaml 
    {{ $k8sMinorVersionToCCMVersion := dict
    "1.27" "v1.27.7"
    "1.28" "v1.28.6"
    "1.29" "v1.29.3"
    "1.30" "v1.30.1"
    }}
```

https://helm.sh/docs/chart_template_guide/function_list/#dict it should be a list of pairs

Reply:
I see. thanks for the reference. I was looking at go template syntax for dict.